### PR TITLE
Release 1.1.2

### PR DESCRIPTION
**Pull request by boxanm**
Follow-up of https://github.com/norlab-ulaval/libnabo/pull/139.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,349 @@
...
+
+# Celery stuff
+celerybeat-schedule
+celerybeat.pid
+
+# SageMath parsed files
+*.sage.py
+
+# Environments
+.env
```
Comment (RedLeader962) [inline .gitignore:281]: @boxanm Recommand to never put `.env` `*.env` or `.env.*` in the gitignore. The `.env` file is the default environment file name that docker compose seek for in a directory and all those variation are used by `dockerized-norlab` and `norlab-build-system`.

**Review by RedLeader962 — CHANGES_REQUESTED** [review]
Good job! Apart from my comment on the `.env` in the gitignore, the rest is good to go.

File: .gitignore (hunk 1 above)
Comment (boxanm) [reply .gitignore]: I removed both `.env` and `env/ `from .gitignore.

**Review by RedLeader962 — APPROVED** [review]
Ready to merge 👍

**Pull request merged**